.github/workflows/finalize-release.yaml

```diff
@@ -8,16 +8,20 @@ name: Finalize Release
 on:
   workflow_dispatch:
   push:
-    tags:
+    branches:
```

Comment:
does this mean anytime a commit is pushed to any release branch itll run this workflow??
or is this only when a commit is pushed to this branch with changes to any chnagelog.md file?

Reply:
Commit to branch, and change to a changelog.md file. These two fields are and'd (according to the GH docs), so both are required for this workflow to run. However, doing something like `tags: ... branches: ...` is or'd, so that was not an option.